src/rHelpPanel.ts

```diff
@@ -392,47 +138,27 @@ export class HelpPanel implements api.HelpPanel {
 		return true;
 	}
 
-	// retrieves the stored webview or creates a new one if the webview was closed
-	private getWebview(): Webview {
-		// create webview if necessary
-		if(!this.panel){
-			const webViewOptions: WebviewOptions & WebviewPanelOptions = {
-				enableScripts: true,
-				enableFindWidget: true
-			};
-			this.panel = window.createWebviewPanel('rhelp', 'R Help', this.viewColumn, webViewOptions);
-
-			// virtual uris used to access local files
-			this.webviewScriptUri = this.panel.webview.asWebviewUri(this.webviewScriptFile);
-			this.webviewStyleUri = this.panel.webview.asWebviewUri(this.webviewStyleFile);
-
-			// called e.g. when the webview panel is closed by the user
-			this.panel.onDidDispose(() => {
-				this.panel = undefined;
-				this.webviewScriptUri = undefined;
-				this.webviewStyleUri = undefined;
-				void commands.executeCommand('setContext', 'r.helpPanel.active', false);
-			});
-
-			// sent by javascript added to the help pages, e.g. when a link or mouse button is clicked
-			this.panel.webview.onDidReceiveMessage((e: any) => {
-				void this.handleMessage(e);
-			});
-
-			// set context variable to show forward/backward buttons
-			this.panel.onDidChangeViewState((e: WebviewPanelOnDidChangeViewStateEvent) => {
-				void commands.executeCommand('setContext', 'r.helpPanel.active', e.webviewPanel.active);
-			});
-
+	public async openInExternalBrowser(helpFile?: HelpFile): Promise<boolean> {
+		if(!this.currentEntry){
+			return false;
 		}
-
-		this.panel.reveal();
-
-		return this.panel.webview;
+		if(!helpFile){
+			helpFile = this.currentEntry.helpFile;
+		}
+		const url = helpFile.url;
+		if(!url){
+			return false;
+		}
+		const uri = vscode.Uri.parse(url);
+		const externalUri = await vscode.env.asExternalUri(uri);
```

Comment:
According to https://code.visualstudio.com/api/advanced-topics/remote-extensions#opening-something-in-a-local-browser-or-application and https://code.visualstudio.com/api/advanced-topics/remote-extensions#forwarding-localhost, `vscode.env.asExternalUri()` is not needed in this case. Directly using `vscode.env.openExternal(uri)` should work.

Reply:
The docs of `asExternalUri` also says:

>Note that uris passed through openExternal are automatically resolved and you should not call asExternalUri on them.

Reply:
Thanks 👍